CHANGELOG.md

```diff
@@ -9,10 +9,8 @@
 ### Fixes
 
 - Make `SentryDiagnosticSubscriber._disposableListeners` thread safe ([#1506](https://github.com/getsentry/sentry-dotnet/pull/1506))
- 
-### Fixes
-
 - Adjust database span names by replacing `_` to `.`. `db.query_compiler` becomes `db.query.compile`. ([#1502](https://github.com/getsentry/sentry-dotnet/pull/1502))
+- Resolve `System.Text.Json` compatibility warning for one test project ([#1509](https://github.com/getsentry/sentry-dotnet/pull/1509))
```

Comment:
given this is only test projects. no changelog required

Reply:
Sure. I was just responding to the CI failure from Danger. 🙂

Reply:
u can bypass that by adding the text `#skip-changelog`  anywhere in the pr contents